use-cosmoz-grouped-list.js

```diff
@@ -109,8 +109,17 @@ const styles = {
 		);
 
 		useEffect(() => {
-			const handler = event =>
-				host.querySelector('#list').updateSizeForIndex(event.detail.index);
+			const handler = e => {
+				let index = e.detail?.index;
+				if (index == null) {
+					const path = e.composedPath();
+					index = path.slice(0, path.indexOf(host))
+						.find(e => e.tagName.toLowerCase() === 'cosmoz-grouped-list-row')?.index;
+				}
+				if (index != null) {
```

Comment:
```suggestion
				} else {
```

Reply:
It's not the same thing as the code above. I re-evaluate index after setting in the if branch,